tests/testthat/helper-initialize.R

```diff
@@ -382,3 +382,11 @@ skip_databricks_connect <- function() {
   if (Sys.getenv("TEST_DATABRICKS_CONNECT") == "true")
     skip("Test is skipped on Databricks Connect")
 }
+
+get_simple_data_path <- function(file_name) {
+    if (Sys.getenv("TEST_DATABRICKS_CONNECT") == "true")
+        sample_data_path <- paste("dbfs:/tmp/data/", file_name, sep="")
+    else
+        sample_data_path <- dir(dir_path, recursive = TRUE, pattern = file_name, full.names = TRUE)
```

Comment:
I think it would be useful to support `dbfs://` when using `method = "databricks"`, not sure what is missing to get this working... probably just add a few more JARs that support read/write from `dbfs://`?

I do second Hossin's comment about not adding `spark_read/write_dbfs()`, since I would expect users will want to use all the read/write functions (`spark_read_csv`, `spark_read_json`, etc) with DBFS.